src/intent_pilot/utils/config.py

```diff
@@ -36,11 +36,6 @@ def __init__(self, verbose=False):
         self.openai_temperature = 0.1
         self.openai_max_tokens = 1000
 
-        self.user_config_dir = pathlib.Path.home() / pathlib.Path(".askui")
-        self.user_config_env_path = self.user_config_dir / "intent-pilot.env"
-
-        # load from user config file
-        load_dotenv(self.user_config_env_path)
```

Comment:
Since we provide the option to use wheel installable, how does .env work in that regard? Users would have to provide .env file in the project folder which in this case would be site-packages/.
Also, if we never save the keys, do they keep entering them every time they use it?

Reply:
Hm, I am not too familiar with the ecosystem but I just tested it and it worked with the local `.env` in the root project folder. I also used a virtual environment. How does it work there?

Basically, users have to reenter 😉 . But they could use the `.env` instead which is the same as saving it in `intent-pilot.env`.

Reply:
Ah, I see. Thanks for the explanation.

What do you think of this implementation of your suggestion: Add back the possibility to save to config but make it optional behind a flag?

Reply:
sounds good. Thanks!

Reply:
@gitlost-murali-askui Can you take a look again?

I added a flag to delete the config file and another one that enables the save prompt.